models/event.js

```diff
@@ -19,6 +19,7 @@ const MODEL = {
     createTime: Joi
         .string()
         .isoDate()
+        .max(32)
```

Comment:
Good question.  I think we were storing it as a String and letting the users parse it.  But we may want to switch to Date.